### PR TITLE
MSEDO-192: Now redirection is correct to details

### PR DESCRIPTION
**Pull request by andrenns**

### CODE REVIEW

**Comment by xalves** [comment]
Looks nice from my perspective!
Just a small comment:
* When we are seeing the details of the therapeutical regime. Should the "voltar" button redirect to the list of therapeutical regimes? (I'm also trying to get that settled to see if we can merge).

**Comment by DanielFran** [comment]
The button "Voltar" should go back to the previous page, so it depends on where you come.

**Review by DanielFran — APPROVED** [review]

File: src/main/webapp/app/entities/therapeutic-regime/therapeutic-regime-update.component.html [hunk 1]
```diff
@@ -1,10 +1,11 @@
-<div class="row justify-content-center">
+<div class="row justify-content-center" >
```
Comment (sfdsilva) [inline src/main/webapp/app/entities/therapeutic-regime/therapeutic-regime-update.component.html:1]: Is this change needed?

File: src/main/webapp/app/entities/therapeutic-regime/therapeutic-regime-update.component.html [hunk 2]
```diff
@@ -1,10 +1,11 @@
...
     <div class="col-8">
         <form name="editForm" role="form" novalidate (ngSubmit)="save()" [formGroup]="editForm">
             <h2 class="list-text-color-bold" id="custom-therapeutic-regime-heading" data-cy="TherapeuticRegimeCreateUpdateHeading" jhiTranslate="supportivecareApp.therapeuticRegime.home.createOrEditLabel">Create or edit a Therapeutic Regime</h2>
             <div class="bg-light" >
                 <h5 class="list-text-color-bold col-lg-12 mt-4 mb-4"  jhiTranslate="supportivecareApp.therapeuticRegime.generalInfo"> >General information</h5>
                 <custom-alert-error></custom-alert-error>
+
```
Comment (sfdsilva) [inline src/main/webapp/app/entities/therapeutic-regime/therapeutic-regime-update.component.html:8]: Is this change needed?

**Review by sfdsilva — APPROVED** [review]

File: src/main/webapp/app/entities/therapeutic-regime/therapeutic-regime-update.component.html (hunk 1 above)
Comment (andrenns) [reply src/main/webapp/app/entities/therapeutic-regime/therapeutic-regime-update.component.html:1]: This change does not affect anything, might be to the fact that we were experimenting and forgot a blank space

File: src/main/webapp/app/entities/therapeutic-regime/therapeutic-regime-update.component.html (hunk 2 above)
Comment (andrenns) [reply src/main/webapp/app/entities/therapeutic-regime/therapeutic-regime-update.component.html:8]: This change does not affect anything, might be to the fact that we were experimenting and forgot a blank space

File: src/main/webapp/app/entities/therapeutic-regime/therapeutic-regime-update.component.html (hunk 1 above)
Comment (sfdsilva) [reply src/main/webapp/app/entities/therapeutic-regime/therapeutic-regime-update.component.html:1]: Yes, I already approved this PR. What I was suggesting is that maybe we can dismiss these changes (not important tho).

**Comment by DanielFran** [comment]
@andrenns @vkhsunny Can you please fix the conflicts to be merged?
